apps/publish/content/correct.py

```diff
@@ -78,6 +80,22 @@ def set_state(self, original, updates):
         else:
             super().set_state(original, updates)
 
+    def change_being_corrected_to_published(self, updates, original):
+        if app.config.get('CORRECTIONS_WORKFLOW') and original.get('state') == 'correction':
+            publish_service = get_resource_service('published')
+            being_corrected_articles = publish_service.find({
```

Comment:
is it actually possible to have more than 1? if not using `find_one` would be better to make it explicit

Reply:
I might have been wrong here, if it's published collection there might be more items with same guid, that would be the case in archive